src/modules/python/pbs/v1/_exc_types.py

```diff
@@ -51,7 +51,7 @@
             'BadResourceValueError'
           ]
 
-class EventIncompatibleError(Exception):
+class EventIncompatibleError(AttributeError):
```

Comment:
Is this change intentional?

Reply:
Yes, we were getting a hasattr error. To fix that, we made EventIncompatibleError class inherit from AttributeError.